_data/localization.json

```diff
@@ -8,111 +8,132 @@
   "title": {
     "en": "Interactive Introduction to Regular Expressions",
     "de": "Interaktive Einführung in Reguläre Ausdrücke",
-    "sv": "Interaktiv Introduktion till Reguljära Uttryck"
+    "sv": "Interaktiv Introduktion till Reguljära Uttryck",
+    "pt-br": "Introdução Interativa para Expressões Regulares"
   },
   "footer": {
     "edit": {
       "en": "improve page",
       "de": "Seite verbessern",
-      "sv": "förbättra sidan"
+      "sv": "förbättra sidan",
+      "pt-br": "melhore a página"
     },
     "newIssue": {
       "en": "report problem",
       "de": "Problem melden",
-      "sv": "rapportera problem"
+      "sv": "rapportera problem",
+      "pt-br": "reporte um problema"
     },
     "deleteCookies": {
       "en": "delete state",
       "de": "Zustand löschen",
-      "sv": "radera tillstånd"
+      "sv": "radera tillstånd",
+      "pt-br": "excluir estado"
     },
     "view": {
       "en": "view source",
       "de": "Quelltext anzeigen",
-      "sv": "visa källa"
+      "sv": "visa källa",
+      "pt-br": "ver código fonte"
     },
     "nextStep": {
       "en": "Next Step",
       "de": "Nächster Schritt",
-      "sv": "Nästa steg"
+      "sv": "Nästa steg",
+      "pt-br": "Próximo passo"
     }
   },
   "hint": {
     "shown": {
       "en": "Hide the hint!",
       "de": "Hinweis verstecken!",
-      "sv": "Göm ledtråden!"
+      "sv": "Göm ledtråden!",
+      "pt-br": "Esconder a dica!"
     },
     "hidden": {
       "en": "Show a hint?",
       "de": "Hinweis anzeigen?",
-      "sv": "Visa en ledtråd?"
+      "sv": "Visa en ledtråd?",
+      "pt-br": "Mostrar uma dica?"
     }
   },
   "quiz": {
     "before": {
       "en": "Regular Expression: ",
       "de": "Regulärer Ausdruck: ",
-      "sv": "Reguljärt uttryck: "
+      "sv": "Reguljärt uttryck: ".
```

Comment:
Thanks for creating the PR and starting the translation effort!
In this line, I believe, there must be a comma "," instead of a dot.